src/routes/page.scss

```diff
@@ -0,0 +1,8 @@
+@use "uswds-core" as * with (
+  $theme-show-notifications: false
+);
+
+.usa-prose > .ldaf-homepage__popular-resources-heading {
+  margin-bottom: 1rem;
+  @include u-font("sans", $theme-h2-font-size);
```

Comment:
Would you mind applying the "sans" font to the card titles while you're here? Just noticed they don't match the mocks (both the featured service cards on the home page and the other cards across the site look like they should be using a sans font heading).

I'm sure there are other places we need to update the fonts but this seems like a safe change to make as a one-off here.